src/composite-data/actions/staker/types.ts

```diff
@@ -0,0 +1,25 @@
+import BigNumber from "bignumber.js"
+import { BigNumber as BigNumberE } from "ethers"
+
+export interface UserStake {
+  amount: BigNumberE
```

Comment:
I wonder if this is a good time to switch the BigNumberE usage to BigNumber

Reply:
Probably a way bigger refactor tho

Reply:
Yeah, I'll do it after 5 of these PR merged

Reply:
This makes sense to me. Would be nice to migrate away from `bignumber.js` and use `ether`'s package, but I'm sure there's a use case I've not considered here. @sandbochs Any insight into that? I feel like we discussed this with Lucky a long while ago, but I can't recall his answer